timed/redmine/management/commands/update_project_expenditure.py

```diff
@@ -87,6 +95,6 @@ def handle(self, *args, **options):
 
             self.stdout.write(
                 self.style.SUCCESS(
-                    f"Updating Redmine issue {project.redmine_project.issue_id} with total spent hours {total_spent_hours}, amount offered {project.amount_offered.amount}, amount invoiced {project.amount_invoiced.amount}"
+                    f"Updating Redmine issue {project.redmine_project.issue_id} with total spent hours {total_spent_hours}, amount offered {amount_offered}, amount invoiced {amount_invoiced}"
```

Comment:
unless it could not have come this far without that attribute..
```suggestion
                    f"Updating Redmine issue {project.redmine_project and project.redmine_project.issue_id} with total spent hours {total_spent_hours}, amount offered {amount_offered}, amount invoiced {amount_invoiced}"
```

Reply:
It can't come that far, if it's not there. see try block L52